README.md

```
@@ -2,35 +2,137 @@
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+```
 
-### docker cli (recommended)
+For ARM64 with an iGPU like Jetson devices:
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)